pypeit/core/procimg.py

```diff
@@ -912,44 +889,39 @@ def trim_frame(frame, mask):
     Trim the masked regions from a frame.
 
     Args:
-        frame (:obj:`numpy.ndarray`):
+        frame (`numpy.ndarray`_):
             Image to be trimmed
-        mask (:obj:`numpy.ndarray`):
+        mask (`numpy.ndarray`_):
             Boolean image set to True for values that should be trimmed
             and False for values to be returned in the output trimmed
             image.
 
     Return:
-        :obj:`numpy.ndarray`:
-            Trimmed image
+        `numpy.ndarray`_: Trimmed image
 
     Raises:
         PypitError:
             Error raised if the trimmed image includes masked values
             because the shape of the valid region is odd.
     """
     # TODO: Should check for this failure mode earlier
-    if np.any(mask[np.invert(np.all(mask,axis=1)),:][:,np.invert(np.all(mask,axis=0))]):
+    if np.any(mask[np.logical_not(np.all(mask,axis=1)),:][:,np.logical_not(np.all(mask,axis=0))]):
         msgs.error('Data section is oddly shaped.  Trimming does not exclude all '
                    'pixels outside the data sections.')
-    return frame[np.invert(np.all(mask,axis=1)),:][:,np.invert(np.all(mask,axis=0))]
+    return frame[np.logical_not(np.all(mask,axis=1)),:][:,np.logical_not(np.all(mask,axis=0))]
 
 
-def variance_frame(datasec_img, sciframe, gain, ronoise, darkcurr=None,
-                   exptime=None, skyframe=None, objframe=None, adderr=0.01, rnoise=None):
+def variance_frame(datasec_img, sciframe, gain, ronoise, darkcurr=None, exptime=None,
+                   skyframe=None, objframe=None, adderr=0.01, rnoise=None):
     """
     Calculate the variance image including detector noise.
 
-    .. todo::
-        This needs particular attention because exptime and darkcurr;
-        used to be dnoise
-
     Args:
-        datasec_img (:obj:`numpy.ndarray`):
+        datasec_img (`numpy.ndarray`_):
             Image that identifies which amplifier (1-indexed) was used
             to read each pixel.  Anything less than 1 is ignored.
-        sciframe (:obj:`numpy.ndarray`):
-            Science frame with counts in ?
+        sciframe (`numpy.ndarray`_):
+            Science frame with counts in electrons.
```

Comment:
Please add the units somewhere in the docs so it is clear this is all in electrons, and ronoise must be in electrons.

Reply:
Done in the basic image processing doc (image_proc.rst), but please check me and note if there are additional places we should add units.